### ISSUE TITLE
README should guide users wrt the ekg eco-system

### ISSUE THREAD

**Issue by alexanderkjeldaas** [tibbe/ekg#69]
There are various ekg packages on hackage, for servant, wai, prometheus.  Sometimes multiple implementations that seem to overlap, for example for wai.

Some sort of overview of this eco-system should exist somewhere, preferably in the ekg package, and preferably in the README which is the first entry-point for new users.

**Comment by tibbe** [comment]
Good idea. I'd start with documenting the core packages and their relationships and then add some pointers to various well-used packages.
